### PR TITLE
R21.1 internet connection error help doc

### PR DESCRIPTION
**Pull request by AijunHall**
Heres what the message looks like now.

image

"Please check to ensure you have an internet connection. If you do have an internet connection, and are still receiving this error, please refer to _support_ or alternatively _contact_ us."

### CODE REVIEW

File: SmartSlicePlugin/plugin.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "SmartSlice",
     "id" : "SmartSlicePlugin",
     "author": "Teton Simulation, Inc.",
-    "version": "21.1.10",
```
Comment (AijunHall) [inline SmartSlicePlugin/plugin.json:5]: Forgot to increment the version for the `plugin.json` on my last ticket. It should be `r21.1.12` now to align with the `package.json`

File: SmartSlicePlugin/SmartSliceAPI.py [hunk 2]
```diff
@@ -496,9 +498,10 @@ def _handleThorErrors(self, http_error_code, returned_object):
             self._error_message.setText(i18n_catalog.i18nc("@info:status", "SmartSlice Server Error (429: Too Many Attempts)"))
         elif http_error_code == self.ConnectionErrorCodes.genericInternetConnectionError:
             self._error_message.setText(i18n_catalog.i18nc(
-                "@info:status", "Unable to contact SmartSlice servers!<br><br>Please check to ensure you have an internet connection. If you"
-                "<br>do have an internet connection, and are still receiving this<br>error, please <A HREF='" + self._url_handler.helpEmail +
-                "?subject=SmartSlice Cannot Connect Error'>contact us</A>."
+                "@info:status", "Unable to contact SmartSlice servers!<br><br>Please check to ensure you have an internet connection. If you "
+                "do have an internet connection, and are still receiving this<br>error, please refer to "
+                "<A HREF='" + self._url_handler.errorGuide + "#unable_to_contact_servers'>support</A> or alternatively "
+                "<A HREF='" + self._url_handler.helpEmail + "?subject=SmartSlice Cannot Connect Error'>contact us</A>."
```
Comment (dougkenik) [inline SmartSlicePlugin/SmartSliceAPI.py:504]: I would remove "contact us" - some people's default email client doesn't open correctly.

File: SmartSlicePlugin/SmartSliceAPI.py [hunk 3]
```diff
@@ -496,9 +498,10 @@ def _handleThorErrors(self, http_error_code, returned_object):
             self._error_message.setText(i18n_catalog.i18nc("@info:status", "SmartSlice Server Error (429: Too Many Attempts)"))
         elif http_error_code == self.ConnectionErrorCodes.genericInternetConnectionError:
             self._error_message.setText(i18n_catalog.i18nc(
-                "@info:status", "Unable to contact SmartSlice servers!<br><br>Please check to ensure you have an internet connection. If you"
-                "<br>do have an internet connection, and are still receiving this<br>error, please <A HREF='" + self._url_handler.helpEmail +
-                "?subject=SmartSlice Cannot Connect Error'>contact us</A>."
+                "@info:status", "Unable to contact SmartSlice servers!<br><br>Please check to ensure you have an internet connection. If you "
+                "do have an internet connection, and are still receiving this<br>error, please refer to "
+                "<A HREF='" + self._url_handler.errorGuide + "#unable_to_contact_servers'>support</A> or alternatively "
```
Comment (dougkenik) [inline SmartSlicePlugin/SmartSliceAPI.py:503]: I would point users here instead.
https://help.tetonsim.com/internet-connectivity

Add this URL to the `URLHandler ` class

**Review by dougkenik — CHANGES_REQUESTED** [review]

**Comment by AijunHall** [comment]
New message

image

**Review by dougkenik — APPROVED** [review]

**Review by bradymadams — APPROVED** [review]

**Pull request merged**